controllers/admin/AdminModulesController.php

```diff
@@ -124,10 +124,10 @@ public function __construct()
     public function checkCategoriesNames($a, $b)
     {
         if ($a['name'] === $this->trans('Other Modules')) {
-            return true;
+            return 0;
```

Comment:
This is also a BC break, as this function was returning boolean values, and is now returning integer.

Reply:
that's the whole point as the uasort that calls this function expects an integer and not a boolean.

Reply:
yes, but this function is returning a boolean, and cannot return now a integer (because of BC Break). The deprecation comes from the function usage, not the function itself. There would be no problem if the function was private, but as it is public, we must protect usages for other developers using that function for other purposes.

Reply:
@FabienPapet
Is such a strict BC break policiy appropriate here?
If we cannot merge this PR because of a BC break, PS 8.0 does not support PHP 8.0. If PS 8.0 is supposed to support PHP 8.0, we must merge this PR as soon as possible.

IMHO, this is a issue that was missed when PS was made compatible with PHP 8.0. Or am I missing something?

Reply:
it was at the time the pr was written, because of the bc break promise for 8.1 . Now that the next version is 9.0, such a change should be allowed.